ergo/distributions/conditions.py

```diff
@@ -57,3 +72,38 @@ def loss(self, dist):
 
     def __str__(self):
         return f"The probability density function looks similar to the provided density function."
+
+
+@dataclass
+class IntervalCondition(Condition):
+    """
+    Condition that the specified interval should include
+    as close to the specified probability mass as possible
+
+    :raises ValueError: max must be strictly greater than min
+    """
+
+    p: float
+    min: float
```

Comment:
I would reorder to `min, max, p` here, in `__init__`, and in elicit call. Defining bounds of interval first feels a bit more readable to me. Also wondering if it should be `xmin` and `xmax` but feel unsure.

Reply:
I put p first because it's required/has no default, since there is no obvious default

I'm not a huge fan of `xmin` `xmax`

I'd be into `lower_bound` `upper_bound`

I'm going to leave this as I had it for now, but we can change it later